dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -280,11 +307,29 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return wil::str_printf<std::wstring>(L"%ws%ws%ws", m_inlineImage.c_str(), m_heroImage.c_str(), m_appLogoOverride.c_str());
     }
 
-    std::wstring AppNotificationBuilder::GetButtons()
+    std::wstring AppNotificationBuilder::GetActions()
     {
-        if (m_buttonList.size())
+        if (m_buttonList.empty() && m_textBoxList.empty() && m_comboBoxList.empty())
+        {
+            return {};
+        }
+        else
         {
             std::wstring result{};
+            for (auto input : m_textBoxList)
+            {
+                auto placeHolderTextAndTitle = input.hasPlaceHolderTextAndTitle
```

Comment:
Recommend moving this element construction to AddTextBox(...)

Reply:
done